multisite/threadlocals.py

```diff
@@ -41,7 +43,7 @@ def __init__(self, default=None, *args, **kwargs):
         ``default``, if specified, determines the default SITE_ID,
         if that is unset.
         """
-        if default is not None and not isinstance(default, (int, long)):
```

Comment:
These and the following changes will not work the same way on Python 2.7 either.